test/rpc_test.py

```diff
@@ -16,13 +16,13 @@
 from torch.distributed.rpc.internal import PythonUDF, _internal_rpc_pickler
 
 
-def requires_process_group_agent(func):
-    from torch.distributed.rpc.api import _agent
-
-    return unittest.skipUnless(
-        isinstance(_agent, ProcessGroupAgent),
-        "Only ProcessGroupAgent supports global termination detection",
-    )
+def requires_process_group_agent(message=""):
```

Comment:
Does this decorator run tests for you? I found it still skips tests for me, and the following works:

```
def requires_process_group_agent(func):
    from torch.distributed.rpc.api import _agent

    return unittest.skipUnless(
        TEST_CONFIG.rpc_backend == RpcBackend.PROCESS_GROUP,
        "Only ProcessGroupAgent supports global termination detection",
    )(func)
```

Reply:
@mrshenli 

It's working for me.